app/helper/RTMediaSettings.php

```diff
@@ -234,7 +234,6 @@ public function settings() {
 				$http_referer = rtm_get_server_var( 'HTTP_REFERER', 'FILTER_SANITIZE_URL' );
 				if ( isset( $http_referer ) ) {
 					wp_safe_redirect( $http_referer . $settings_saved );
-					exit();
```

Comment:
Note: wp_safe_redirect() does not exit automatically, and should almost always be followed by a call to exit; 

Can we check if it's any other issue because wp_safe_redirect needs to have exit, or we can put exit conditionally if it's successfully redirected.

Reply:
@ravatparmar I understand what you're saying, I read the doc. I tried many things to add `exit()` but it just doesn't work that way. There is a code running after this function so I can't add `exit()` in this function. So I think the code was written this way.
`exit()` was added as phpcs fix, before adding it was working fine.
I think we should keep it this way because this function is common for all rtMedia settings. Making change here might break other things.

Reply:
As you said here `There is a code running after this function`, if possible can we move this redirection at end of the function? so code will be executed as normal and then it'll redirected to the next specific page.

Reply:
I already tried moving it at the end of the function, not working.

Reply:
Okay, one last thing, before `wp_safe_redirect( $http_referer . $settings_saved );`, can you please write down comment why we are not putting exit after `wp_safe_redirect` with the issue so later it can be remembered and don't get this issue again.

Reply:
Added the comment